core/dovecot/start.py

```diff
@@ -32,6 +33,15 @@ def start_podop():
 for dovecot_file in glob.glob("/conf/*.conf"):
     conf.jinja(dovecot_file, os.environ, os.path.join("/etc/dovecot", os.path.basename(dovecot_file)))
 
+try:
```

Comment:
Makedirs sounds more fitted for this and will remove the need for exception handling.

Reply:
you are, of course, right. Thanks, I didn’t know makedirs had `exist_ok` …